jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -936,6 +983,63 @@ class SingleProcessingStore extends Reflux.Store {
       this.isProcessingDataLoaded
     );
   }
+
+  getDisplays() {
+    return this.displays;
+  }
+
+  refreshDisplays() {
+    const tab = this.getActiveTab();
+
+    // Tab specific displays
+    if (tab === SingleProcessingTabs.Translations) {
+      this.displays.set(StaticDisplays.Transcript, true);
+      this.clearTranslationDisplays();
+    } else if (tab === SingleProcessingTabs.Transcript) {
+      this.displays.delete(StaticDisplays.Transcript);
+      this.populuateTranslationDisplays();
+    } else {
+      this.populuateTranslationDisplays();
+      // If transcript display is on (coming from translations) then leave it on.
+      this.displays.set(
+        StaticDisplays.Transcript,
+        this.displays.get(StaticDisplays.Transcript) || false
+      );
+    }
+  }
+
+  setStaticDisplay(display: StaticDisplays) {
+    switch (display) {
+      case StaticDisplays.Audio:
+        this.displays.set(
+          StaticDisplays.Audio,
+          !this.displays.get(StaticDisplays.Audio)
+        );
+        break;
+      case StaticDisplays.Data:
+        this.displays.set(
+          StaticDisplays.Data,
+          !this.displays.get(StaticDisplays.Data)
+        );
+        break;
+      case StaticDisplays.Transcript:
+        this.displays.set(
+          StaticDisplays.Transcript,
+          !this.displays.get(StaticDisplays.Transcript)
+        );
+        break;
+    }
```

Comment:
Do we need the switch case statement here? Would TS allow simple `this.displays.set(display, isEnabled)`?